src/bootstrap/builder/tests.rs

```diff
@@ -8,9 +8,9 @@ fn configure(cmd: &str, host: &[&str], target: &[&str]) -> Config {
     config.save_toolstates = None;
     config.dry_run = true;
     config.ninja_in_file = false;
-    config.out = PathBuf::from(env::var_os("BOOTSTRAP_OUTPUT_DIRECTORY").unwrap());
-    config.initial_rustc = PathBuf::from(env::var_os("RUSTC").unwrap());
-    config.initial_cargo = PathBuf::from(env::var_os("BOOTSTRAP_INITIAL_CARGO").unwrap());
+    config.out = PathBuf::from(env!("OUT_DIR"));
```

Comment:
ah, good point, cargo will probably cache these temporary files indefinitely. One alternative is to just use the `out` determined by `Config::parse`? But unfortunately cargo runs these tests with a working directory of `src/bootstrap`, so it will use `src/bootstrap/build` instead of the top-level build file :( you mentioned in another PR adding `tempdir` seems overkill, so I think `OUT_DIR` is probably an OK alternative? at least, it shouldn't hurt anything, nothing else in bootstrap reads from OUT_DIR.